x-pack/plugins/cases/public/containers/use_get_case_user_actions.tsx

```diff
@@ -54,55 +57,20 @@ export interface UseGetCaseUserActions extends CaseUserActionsState {
   ) => Promise<void>;
 }
 
-const unknownExternalServiceConnectorId = 'unknown';
-
-const getExternalService = (
-  connectorId: string | null,
-  encodedValue: string | null
-): CaseExternalService | null => {
-  const decodedValue = parseStringAsExternalService(connectorId, encodedValue);
-
-  if (decodedValue == null) {
-    return null;
-  }
-  return {
-    ...convertToCamelCase<CaseFullExternalService, CaseExternalService>(decodedValue),
-    // if in the rare case that the connector id is null we'll set it to unknown if we need to reference it in the UI
-    // anywhere. The id would only ever be null if a migration failed or some logic error within the backend occurred
-    connectorId: connectorId ?? unknownExternalServiceConnectorId,
-  };
-};
-
 const groupConnectorFields = (
   userActions: CaseUserActions[]
 ): Record<string, Array<CaseConnector['fields']>> =>
   userActions.reduce((acc, mua) => {
-    if (mua.actionField[0] !== 'connector') {
-      return acc;
-    }
-
-    const oldConnector = parseStringAsConnector(mua.oldValConnectorId, mua.oldValue);
-    const newConnector = parseStringAsConnector(mua.newValConnectorId, mua.newValue);
+    if (isConnectorUserAction(mua) || isCreateCaseUserAction(mua)) {
+      const connector = mua.payload.connector;
 
-    if (!oldConnector || !newConnector) {
-      return acc;
+      return {
+        ...acc,
+        [connector.id]: [...(acc[connector.id] || []), connector.fields],
```

Comment:
Probably not because the fields of the `none` connector are always the same. Nevertheless, I think we should return the `acc` if the `connector.id === none`